recipe/meta.yaml

```diff
@@ -70,7 +70,7 @@ source:
     sha256: de3c87b26a80e789986d8e6950c6304175d3829afe9c6c7211eb7257266ab0ac         # [win]
 
 build:
-  number: 100
+  number: 0
```

Comment:
```suggestion
  number: 100
```

Reply:
Why do we need this build number Isuru?

Reply:
To fix some issues where pypy is prioritised.

Reply:
Ah ok. Do we need this for Python 3.6 as well?